Fabulous.XamarinForms/src/Fabulous.XamarinForms.Core/ViewHelpers.fs

```diff
@@ -168,3 +168,9 @@ module ViewHelpers =
                 let res = ViewElement(externalObj.GetType(), create, update, attribs)
                 externalsTable.Add(externalObj, res)
                 res
+
+    // Helper for checking Platform - especially ShellHasNavBarShadow
+    let updateShellNavBarHasShadowHelper = 
```

Comment:
I would directly write the "match ... with" instruction inside `Shell.SetNavBarHasShadow` on line 684 of ViewUpdaters.fs
You can ignore the `defaultValue` field in the bindings file since the `updateCode` field is provided.

Reply:
Okay didn't know that the defaultValue is not needed if updateCode is there. will change it right now

Reply:
Oh. Apparently, I coded CodeGen to require the field even if unused.
Should fix that later. For now, only put an empty value for `defaultValue`.

Also I've noticed a few warnings from CodeGen

>Property 'IndicatorLayout' on type 'IndicatorView' not found
Property 'IndicatorShape' on type 'IndicatorView' not found
Property 'Item' on type 'SwipeItems' not found

Reply:
I can understand why `item`may not be found: 
`public virtual ISwipeItem Item { get; set; }`

Don't know about the Indicatorproperties. Looking good to me or maybe I need to specify the type? https://docs.microsoft.com/en-us/xamarin/xamarin-forms/release-notes/4.4/4.4.0-api

Reply:
`IndicatorShape` is missing an 's'. It's `IndicatorsShape`.

`IndicatorLayout` bindable property is not marked as public. Instead, it appears we need to use `IndicatorTemplate`. So it can be removed from the binding.
https://github.com/xamarin/Xamarin.Forms/blob/bbb07f6f0ff533299a6fa183bc86e6be88ceecb0/Xamarin.Forms.Core/IndicatorView.cs#L36

And for some reasons, `Item` of SwipeItems is on the release notes, but not in the source code...

Reply:
In jsut removed those and thanks for the hint of the missing s :D